boards/tsi/ek_tsi_skyp/ek_tsi_skyp.yaml

```diff
@@ -0,0 +1,19 @@
+identifier: ek_tsi_skyp
+name: TSI sky-p
+type: mcu
+arch: arm
+ram: 2016
+flash: 32
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - gpio
```

Comment:
Since we have not enabled gpio, do you need this ?

Reply:
I will clean up the code and provide a new update.